cddl_gen/cddl_gen.py

```diff
@@ -2353,12 +2407,19 @@ def parse_args():
         formatter_class=RawDescriptionHelpFormatter)
 
     code_parser.add_argument(
-        "--output-c", "--oc", required=True, type=FileType('w'), help="Path to output C file.")
+        "--output-c", "--oc", required=False, type=str, help="Path to output C file.")
     code_parser.add_argument(
-        "--output-h", "--oh", required=True, type=FileType('w'), help="Path to output header file.")
+        "--output-h", "--oh", required=False, type=str, help="Path to output header file.")
     code_parser.add_argument(
-        "--output-h-types", "--oht", required=False, type=FileType('w'),
+        "--output-h-types", "--oht", required=False, type=str,
         help="Path to output header file with typedefs (shared between decode and encode).")
+    code_parser.add_argument(
+        "--copy-sources", required=False, action="store_true", default=False,
+        help="""Copy the non generated source files into the same directories as the
+generated files.""")
+    code_parser.add_argument(
+        "--output-cmake", required=False, type=str,
+        help="Path to output Cmake file.")
```

Comment:
here we can add a note that this will be the name of the target created in the cmake file